### DISCUSSION TITLE
banks

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#1227] [General]
what banks are you using guys?
Need to change mine and looking forward to some recommendations

**Comment by MirkosMangle** [comment]
I currently use a mix of banks, including Bank of USA, and I’ve had a good experience with them. For everyday banking, It offers great customer service and user-friendly online banking. If you’re looking for savings accounts, Texas bank has competitive interest rates.

**Comment by ellielo** [comment]
It really depends on your priorities—like whether you need low fees, high interest rates, or specific services. I’d suggest reading some reviews and possibly visiting a few banks in person to see which one suits your needs best. What features are most important to you in a new bank? I personally find [broadway bank customer service](https://www.pissedconsumer.com/company/broadway-bank/customer-service.html) not bad. They helped me couple of times .
